src/App.js

```diff
@@ -1,15 +1,111 @@
 import React from 'react';
 import './App.css';
+import ToDoList from './Components/ToDoList/ToDoList';
+import { getTodos } from './api/getTodos';
+import { getUsers } from './api/getUsers';
 
-import todos from './api/todos';
-import users from './api/users';
+function getTodosWithUsers(todoArr, userArr) {
+  return todoArr.map(todo => ({
+    ...todo,
+    user: userArr.find(item => item.id === todo.userId),
+  }));
+}
+
+class App extends React.Component {
+  state = {
+    todos: [],
+    users: [],
+    isLoading: false,
+    hasError: false,
+  };
+
+  sortByUser = () => {
+    this.setState(prevState => ({
+      todos: [...prevState.todos].sort((a, b) => (a.user.localCompare(b.user) === 1 ? 1 : -1)),
```

Comment:
`user` это объект, `localCompare` используется для сравнения строковых значений
```suggestion
      todos: [...prevState.todos]
        .sort((a, b) => a.user.name.localCompare(b.user.name)),
```